src/appmixer/wiz/core/UploadSecurityScan/inputs.vulnerabilityFindings.js

```
@@ -0,0 +1,176 @@
+    //         },
+    //         externalFindingLink: {
+    //             type: 'text',
+    //             index: 5,
+    //             label: 'External Finding Link',
+    //             tooltip: 'A link to the source of the external finding.'
+    //         },
+    //         detailedName: {
+    //             type: 'text',
+    //             index: 6,
+    //             label: 'Detailed Name',
+    //             tooltip: 'The details of the externalDetectionSource, such as "Package," should include relevant information about the package. For instance, if the externalDetectionSource is "libncurses6," the "Details Name" should reflect details about the package, such as "libncurses6 package.".'
```

ESLint

[error] This line has a comment length of 303. Maximum allowed is 150.

(max-len)